recipes/hpp-environments/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+mkdir build
+cd build
+cmake .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_CXX_FLAGS=-std=c++17 \
```

Comment:
Do you need this line? It should be in conda-forge's default C++ flags:
```suggestion
```

Reply:
The default value is c++14 for osx and i  that helped me to fix some osx issue to force 17 standard, is it ok ?